library/Requests.php

```diff
@@ -173,12 +173,12 @@ public static function add_transport($transport) {
 	}
 
 	/**
-	 * Get a working transport
+	 * Get the FQCN for a working transport
 	 *
-	 * @throws Requests_Exception If no valid transport is found (`notransport`)
-	 * @return Requests_Transport
+	 * @return string FQCN of the transport to use, or an empty string if none
+	 *                provided the requested capabilities.
 	 */
```

Comment:
Please add a `@param` tag to the docblock and describe the format of the `$capabilities` array parameter.